tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -665,49 +684,15 @@ def test_stride(self):
 
         # 0 effective ids Just take the middle one
         output = speech_recognizer({"raw": waveform, "stride": (5000, 5000), "sampling_rate": 16_000})
-        self.assertEqual(output, {"text": "B"})
+        self.assertEqual(output, {"text": ""})
 
         # Only 1 arange.
         output = speech_recognizer({"raw": waveform, "stride": (0, 9000), "sampling_rate": 16_000})
-        self.assertEqual(output, {"text": "O"})
+        self.assertEqual(output, {"text": "OB"})
 
         # 2nd arange
         output = speech_recognizer({"raw": waveform, "stride": (1000, 8000), "sampling_rate": 16_000})
-        self.assertEqual(output, {"text": "B XB"})
-
-
-@require_torch
```

Comment:
Here we delete a lot of tests -> do you think that's fine?

Reply:
Yes, the function being tested is removed and not called anymore.